ISISNeutronMuon/MDANSE
# Change instrument settings of OSIRIS and IRIS

**Description of work**
The instrument parameters have been updated, and one startup error eliminated.

**Fixes**
1. Instrument list now includes OSIRIS and IRIS.
2. The "None" instrument does not trigger an error message on startup.

**To test**
Start the GUI, load a trajectory and pick the Dynamic Incoherent Scattering Function analysis. Try switching between the four entries for OSIRIS and IRIS PG002 and PG004 reflections. New values of resolution and q limits should be visible.
If the GUI is still showing the old values, try "Restore original parameters" in the context menu of the Instrument List.
